pyproject.toml

```diff
@@ -0,0 +1,7 @@
+# See https://setuptools.readthedocs.io/en/latest/build_meta.html
```

Comment:
We didn't use this before and aren't planning on using that build tool. Can this be removed?

Reply:
I copied it from som template do you want me to remove those file altogether

Reply:
Yes, just remove it

Reply:
Done

Reply:
OK